themes/default/content/blog/google-cloud-7-0/index.md

```diff
@@ -0,0 +1,176 @@
+---
+title: "Pulumi Google Cloud Classic 7.0"
+date: 2023-11-05T09:00:00-04:00
+draft: false
+meta_desc: "The Latest for Your Modern Cloud Infrastructure: Pulumi Google Cloud Provider 7.0."
+meta_image: meta.png
+authors:
+  - monica-rodriguez
+tags:
+  - google-cloud
+  - releases
+
+---
+
+The latest major release of the Pulumi Google Cloud Classic Provider is available now! This [7.0 release](https://github.com/pulumi/pulumi-gcp/releases/tag/v7.0.0) contains the latest upstream changes to keep you up-to-date along with a highly requested bug fix, keeping your journey in managing Google Cloud resources fresh and smooth.
+
+The Pulumi Google Cloud Classic provider can be used to provision any of the cloud resources available in the upstream provider. It is part of the suite of [Pulumi official providers](https://www.pulumi.com/docs/intro/cloud-providers/), which means that it is officially maintained and supported by Pulumi. The provider is also open source and available on [GitHub](https://github.com/pulumi/pulumi-gcp) for you to contribute and grow.
+
+<!--more-->
+
+If you are new to Pulumi and would like to get started with the Google Cloud provider, visit any of these pages to jump right in!
+
+- [Getting Started](https://www.pulumi.com/docs/clouds/gcp/get-started/) - A guided walkthrough for creating your first project
+- [Setup & Install](https://www.pulumi.com/registry/packages/gcp/installation-configuration/) - Instructions on installing the Google Cloud provider
+- [How-to guides](https://www.pulumi.com/registry/packages/gcp/how-to-guides/) - Learn how to use the Google Cloud provider to provision specific resources
+- [Templates](https://www.pulumi.com/docs/clouds/gcp/#templates) - Use a quickstart template to create a new project
+- [Pulumi AI](https://www.pulumi.com/ai) - Ask Pulumi AI to create a new project
+
+## Looking back
+
+Since the initial launch of 6.0 almost two years ago, we've shipped over 70 releases to deliver the latest upstream enhancements while also adding many of our own. Here are some of the highlights:
```

Comment:
```suggestion
Since the initial launch of 6.0 almost two years ago, we've shipped over 70 releases to deliver the latest upstream enhancements while also adding many of our own. Here are some of the highlights from these releases:
```